### ISSUE TITLE
Final Project is now open

### ISSUE THREAD

**Issue by vincenzocoia** [vincenzocoia/BAIT509#18]
...and can be found [here](https://github.com/vincenzocoia/BAIT509/blob/master/assessments/final_project/final.md).

See the new file in Connect for your assigned groupings. Please sit with your group for the upcoming class (March 14 -- this is not necessary for other classes).

**Comment by Coding-Cody** [comment]
Hi professor,

Since the train & test split may affect the result somehow, I am wondering whether we should keep exactly the same training set and test set for every model applied in the project?

Cody Xu


@vincenzocoia @vmasrani @mooahmed @rafizaman

**Comment by mooahmed** [comment]
I would use exactly the same split for all the models for fair comparison.
However, I would average the results over different random seeds, e.g. 5,
to get error bars.
